onnxruntime/core/providers/cpu/symbols.txt

```diff
@@ -1,2 +1,3 @@
 OrtGetApiBase
-OrtSessionOptionsAppendExecutionProvider_CPU
\ No newline at end of file
+OrtSessionOptionsAppendExecutionProvider_CPU
+OrtGetWinMLAdapter
```

Comment:
Does WinMLAdapter have a dedicated folder under  onnxruntime/core/providers ? If yes, you may put this line there.

Reply:
There isn't a dedicated folder under onnxruntime/core/providers. I will add one.

Reply:
see: https://github.com/microsoft/onnxruntime/blob/2f79942ca7bbdb625df7db8f04dd954e1b935900/tools/ci_build/gen_def.py#L24